### DISCUSSION TITLE
Network Programming Basics in Ruby

### DISCUSSION THREAD

**Discussion by moonglum** [therubyradar/submissions#149] [Submissions]
[In this post](https://lucas.dohmen.io/posts/2023/06/04/network-programming-basics-in-ruby), I explore the basics of network programming. It includes a hands-on section, opening a socket and using it for communication. I plan to write more entries in this series.

**Comment by cjilbert504** [comment]
@moonglum - Thank you so much for this submission! Amazing article!

**Reply by moonglum** [reply]
Thanks a lot <3 I'll work on the next one in the series 😄
